src/style.css

```diff
@@ -1,3 +1,160 @@
+* {
+  box-sizing: border-box;
+}
+
+h2 {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  vertical-align: baseline;
+  background: transparent;
+  font-size: 100%;
+}
+
+img {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  vertical-align: baseline;
+  background: transparent;
+  font-size: 100%;
+}
+
+button {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  vertical-align: baseline;
+  background: transparent;
+  font-size: 100%;
+}
+
+html {
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+:root {
+  --main-accent: #060b35;
+  --blue-accent: #00acdc;
+  --white-accent: #fff;
+  --secondary: #616070;
+}
+
 body {
+  margin: auto;
+}
+
+.card {
+  margin: auto;
+  padding-left: 16px;
+  padding-right: 16px;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
```

Comment:
It's a bad practice to hardcode the height of the card, the height should depend on the card's content